File: src/shipping/scheduler/src/com/fabrikam/dronedelivery/deliveryscheduler/scheduler/models/invoker/Location.java
package com.fabrikam.dronedelivery.deliveryscheduler.scheduler.models.invoker;

public class Location {
	private double altitude;
	private double latitude;
	private double longitude;

	public double getAltitude() {
		return altitude;
	}

	public void setAltitude(double altitude) {
		this.altitude = altitude;
	}

	public double getLatitude() {
		return latitude;
	}

	public void setLatitude(double latitude) {
		this.latitude = latitude;
	}

	public double getLongitude() {
		return longitude;
	}

	public void setLongitude(double longitude) {
		this.longitude = longitude;
	}
}

File: src/shipping/drone/Model/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Fabrikam.DroneDelivery.Drone.Model
{
    public class Location
    {
        public Location() { }

        public Location(double altitude, double latitude, double longitude)
        {
            Altitude = altitude;
            Latitude = latitude;
            Longitude = longitude;
        }
        public double Altitude { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}
